config.test.yaml

```diff
@@ -35,6 +35,8 @@ default_project: &default_project
           host: http://citoid-beta.wmflabs.org
         events: {}
         purged_cache_control: test_purged_cache_control
+        # Cache control for purged endpoints allowing short-term client caching
+        purged_cache_control_client_cache: test_purged_cache_control_with_client_caching
```

Comment:
This is an odd value to use, but I guess it eliminates the change of satisfying the test by a static cache-control response.